pkg/vpcmodel/groupingGraphical.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package vpcmodel
+
+import (
+	"github.com/np-guard/models/pkg/ipblock"
+)
+
+// for the graphical (html, drawio, svg) representation. In the graph presentation, each node must have all relevant edges.
+// this is not the case in the textual presentation. E.g., a textual presentation may look like:
+// 142.0.64.0/17 ->vsi2
+// 142.0.0.0/16 -> vsi1
+// 0.0.0.0/0 -> vsi3
+// 142.0.64.0/17 should also be connected to vsi2 and vsi3
+// In order to add missing edges, we go over all the endpoints in grouping that present external nodes, and check for containment
+// if external endpoint e1 is contained in external end point e2 then all the "edges" of e2 are added to e1
+func (g *GroupConnLines) consistencyEdgesExternal() {
+	// 1. Gets a map from external endpoints name to their IPs
+	eeNameToIPBlock := getMapToIps(g.GroupedLines)
+	// 2. Gets a map from external endpoints name to their endpoint
+	eeNameToEE := getMapToEPEs(g.GroupedLines)
+	// 3. Gets a map from external endpoint name to all the endpoint it contains
+	containedMap := getContainedEndpointMap(eeNameToIPBlock, eeNameToEE)
+	// 3. Add edges, based on the above map (3)
+	g.addEdgesOfContainingEPs(containedMap)
+}
+
+// gets []*groupedConnLine and returns a map from the string presentation of each endpoint to its ipBlock
+func getMapToIps(grouped []*groupedConnLine) (eeToIPBlock map[string]*ipblock.IPBlock) {
```

Comment:
```suggestion
func getMapToIps(grouped []*groupedConnLine) (map[string]*ipblock.IPBlock) {
```
Naming the returned value doesn't help much when its type is a map (also in other functions in this file)

Reply:
why?